ykcapi/src/lib.rs

```diff
@@ -107,29 +109,73 @@ impl Registers {
     }
 }
 
-/// Parses the stackmap and saved registers from the given address (i.e. the return address of the
+/// Location in terms of basic block index, instruction index, and function name, of a
+/// variable in the AOT module. Mirrors the LLVM struct defined in ykllvmwrap/jitmodbuilder.cc.
+#[derive(Debug)]
+#[repr(C)]
+struct AOTVar {
+    bbidx: usize,
+    instridx: usize,
+    fname: *const i8,
+}
+
+/// Address and size of a vector containing AOTVars. Mirrors the struct defined in
+/// ykllvmwrap/jitmodbuilder.cc.
+#[derive(Debug)]
+#[repr(C)]
+pub struct AOTMap {
+    addr: *const c_void,
+    size: usize,
```

Comment:
Is `size` the size in bytes or the length in terms of ... ?

Reply:
Bytes.

Reply:
Ok, let's document that.

Reply:
I was talking nonsense. This is actually the length. Fixed in fb19d6c